### PR TITLE
docs(gen): content harmonization

### PR DESCRIPTION
**Pull request by jcirinosclwy**
### Description

- Harmonization of some inconsistent terms.
- Fixed some typos and errors.

### CODE REVIEW

File: faq/objectstorage.mdx [hunk 1]
```diff
@@ -20,7 +20,7 @@ Scaleway provides an integrated UI in the [console](https://console.scaleway.com
 - batch deletion is limited to 1000 objects.
 - empty files are not reported as empty folders.
 
-We provide an Amazon Amazon S3-compatible API for programmatic access or usage with any compatible software. Therefore, we recommend using dedicated tools such as `s3cmd` to manage large data sets.
+We provide an Amazon S3-compatible API for programmatic access or usage with any compatible software. Therefore, we recommend using dedicated tools such as `s3cmd` to manage large data sets.
```
Comment (nerda-codes) [inline faq/objectstorage.mdx:23]: to confirm with @SamyOubouaziz I think this wording has to do with copyrights

File: tutorials/stop-outgoing-spam/index.mdx [hunk 2]
```diff
@@ -14,13 +14,13 @@ dates:
...
 
 ### Understanding what is going on
 
-When you receive an abuse alert for SPAM on one of your machines, an intruder likely gained access to it by either guessing your passwords or by using a security hole that may be present in one of your applications.
+When you receive an abuse alert for spam on one of your machines, an intruder likely gained access to it by either guessing your passwords or by using a security hole that may be present in one of your applications.
 
-Your immediate action is required in this case to stop the SPAMs outgoing from the machine. It is recommended to boot the machine in **rescue mode**, to interrupt the outgoing connections as a first step, before proceeding with more in-depth investigations into what is going on:
+Your immediate action is required in this case to stop the spams outgoing from the machine. It is recommended to boot the machine in **rescue mode**, to interrupt the outgoing connections as a first step, before proceeding with more in-depth investigations into what is going on:
```
Comment (nerda-codes) [inline tutorials/stop-outgoing-spam/index.mdx:23]:
```suggestion
Your immediate action is required in this case to stop the spam outgoing from the machine. It is recommended to boot the machine in **rescue mode**, to interrupt the outgoing connections as a first step, before proceeding with more in-depth investigations into what is going on:
```

File: dedibox-network/rpn/how-to/use-rpn-san-proxmox.mdx [hunk 3]
```diff
@@ -79,7 +79,7 @@ Proxmox requires the Open-iSCSI software to support mounting of iSCSI volumes. I
 
 ## Discover and connect to targets in Proxmox
 
-1. In the **Server View**, click **Datacenter**, then **Storage**.
+1. In the **Server View**, click **Data center**, then **Storage**.
```
Comment (RoRoJ) [inline dedibox-network/rpn/how-to/use-rpn-san-proxmox.mdx:82]: If this is referring to the way it is written in another interface, we should maybe leave it the way it is?

File: dedibox/dedicated-servers/how-to/order-a-server.mdx [hunk 4]
```diff
@@ -23,17 +23,17 @@ categories:
 1. Click **Order** on your Dedibox console homepage.
 2. Click **Server**. The server offers page displays.
     <Message type="note">
-      You can view all servers available in your preferred datacenter by clicking the **Datacenter selection** drop-down menu in the top right corner.
+      You can view all servers available in your preferred data center by clicking the **Data center selection** drop-down menu in the top right corner.
```
Comment (RoRoJ) [inline dedibox/dedicated-servers/how-to/order-a-server.mdx:26]: same comment as above

File: dedibox/dedicated-servers/quickstart.mdx [hunk 5]
```diff
@@ -25,17 +25,17 @@ categories:
 1. Click **Order** on your Dedibox console homepage.
 2. Click **Server**. The server offers page displays.
     <Message type="note">
-      You can view all servers available in your preferred datacenter by clicking the **Datacenter selection** drop-down menu in the top right corner.
+      You can view all servers available in your preferred data center by clicking the **Data center selection** drop-down menu in the top right corner.
```
Comment (RoRoJ) [inline dedibox/dedicated-servers/quickstart.mdx:28]: same here

File: environmental-footprint/environmental-footprint/additional-content/environmental-footprint-calculator.mdx [hunk 6]
```diff
@@ -19,45 +19,45 @@ The environmental footprint estimation tool is currently in private beta. Sign u
...
+Our Methodology is based on the [ADEME (French Government Agency for Ecological Transition) Product Category Rules (PCR) for Data center and Cloud services](https://librairie.ademe.fr/produire-autrement/6105-methodological-standard-for-the-environmental-assessment-of-datacenter-it-hosting-services-and-cloud-services.html), which provide a standardized framework for calculating and reporting the environmental impact of data centers and cloud services. These rules are designed to ensure consistency, transparency, and comparability in the assessment of environmental footprints across the industry.
```
Comment (RoRoJ) [inline environmental-footprint/environmental-footprint/additional-content/environmental-footprint-calculator.mdx:26]: Same here, when referring to the title of the document maybe we should leave it the way they write it (Datacenter)

File: tutorials/create-esxi-cluster-dedibox/index.mdx [hunk 7]
```diff
@@ -256,34 +247,34 @@ vCenter is a web-based tool for graphical management of the entire fabric of dat
 
 1. Open a web browser and point it to the domain name of your vCenter. Log in using the `administrator@vsphere.local` user and the password set during the installation. The vSphere dashboard displays:
     <Lightbox src="scaleway-vsphere_client.webp" width="800" alt="" />
-2. Click **Actions** > **New Datacenter** to create a new datacenter. A datacenter is a group of machines.
-3. Enter a name for the new datacenter and click **OK**. It appears in the list on the left.
-4. Enter the datacenter and click **Actions** > **New Host**. Enter the IP address of your first Dedibox server in the form and click **Next**.
+2. Click **Actions** > **New data center** to create a new data center. A data center is a group of machines.
```
Comment (RoRoJ) [inline tutorials/create-esxi-cluster-dedibox/index.mdx:250]: Same here

File: tutorials/create-esxi-cluster-dedibox/index.mdx [hunk 8]
```diff
@@ -256,34 +247,34 @@ vCenter is a web-based tool for graphical management of the entire fabric of dat
...
 Repeat the import for the second server to manage both of them centrally from the vCenter Server.
 
-## Creating a Cluster
+## Creating a cluster
 
-A cluster is a group of several machines. When a host is added to a cluster, the resources of the host become part of the resources of the cluster, and all resources of the hosts within it are managed by the cluster.
+A cluster is a group of several machines. When a host is added to a cluster, its resources become part of the cluster, and all hosts' resources within it are managed by the cluster.
 
-1. From the **Datacenter** dashboard in your vSphere web interface click **Actions** > **New Cluster**:
+1. From the **Data center** dashboard in your vSphere web interface click **Actions** > **New Cluster**:
```
Comment (RoRoJ) [inline tutorials/create-esxi-cluster-dedibox/index.mdx:273]: Same here

File: tutorials/load-overflow-proxmox-and-elastic-metal/index.mdx [hunk 9]
```diff
@@ -45,19 +45,19 @@ Finally, we can introduce and configure a Load Balancer to evenly distribute tra
 
 ## Preparing Proxmox
 
-1. Log into the Proxmox web interface and select your server in the datacenter view. Then click **Network** in the menu.
+1. Log into the Proxmox web interface and select your server in the data center view. Then click **Network** in the menu.
```
Comment (RoRoJ) [inline tutorials/load-overflow-proxmox-and-elastic-metal/index.mdx:48]: same here

File: tutorials/load-overflow-proxmox-and-elastic-metal/index.mdx [hunk 10]
```diff
@@ -45,19 +45,19 @@ Finally, we can introduce and configure a Load Balancer to evenly distribute tra
...
 2. Click **Create** > **Linux Bridge**. The configuration wizard displays. Enter a private IP for your Elastic Metal server in CIDR format. The **Bridge port** is the name of your primary network interface, followed by a dot and your VLAN-ID. Tick the boxes **Autostart** and **VLAN aware**, then click **Create** to create the bridge.
     <Lightbox src="scaleway-px-create-linux-bridge.webp" alt="" />
 3. Click **Apply configuration** to activate the new configuration.
-4. Go to the **Datacenter** view and click **Create Cluster**.
+4. Go to the **Data center** view and click **Create Cluster**.
```
Comment (RoRoJ) [inline tutorials/load-overflow-proxmox-and-elastic-metal/index.mdx:52]: same here

File: tutorials/load-overflow-proxmox-and-elastic-metal/index.mdx [hunk 11]
```diff
@@ -45,19 +45,19 @@ Finally, we can introduce and configure a Load Balancer to evenly distribute tra
...
 5. Enter a name for the cluster and select your Private Network bridge `vmbr1` as the cluster network. Then click **Create** to launch cluster creation.
 6. Select the cluster in the cluster list and click **Join Information**. The cluster join information displays.
 7. Click **Copy Information** to copy the cluster join information into the clipboard.
-8. Go to the Proxmox interface of your second Elastic Metal server and click **Cluster** to enter the cluster configuration from the Datacenter view of the machine.
+8. Go to the Proxmox interface of your second Elastic Metal server and click **Cluster** to enter the cluster configuration from the Data center view of the machine.
```
Comment (RoRoJ) [inline tutorials/load-overflow-proxmox-and-elastic-metal/index.mdx:56]: same here

**Review by RoRoJ — APPROVED** [review]

File: dedibox-network/rpn/how-to/use-rpn-san-proxmox.mdx (hunk 3 above)
Comment (jcirinosclwy) [reply dedibox-network/rpn/how-to/use-rpn-san-proxmox.mdx:82]: Well spotted. I'll fix it. Thanks, Ro!

Comment (jcirinosclwy) [inline dedibox-network/rpn/how-to/use-rpn-san-proxmox.mdx:82]:
```suggestion
1. In the **Server View**, click **Datacenter**, then **Storage**.
```

File: dedibox/dedicated-servers/how-to/order-a-server.mdx (hunk 4 above)
Comment (jcirinosclwy) [inline dedibox/dedicated-servers/how-to/order-a-server.mdx:26]:
```suggestion
      You can view all servers available in your preferred data center by clicking the **Datacenter selection** drop-down menu in the top right corner.
```

File: dedibox/dedicated-servers/quickstart.mdx (hunk 5 above)
Comment (jcirinosclwy) [inline dedibox/dedicated-servers/quickstart.mdx:28]:
```suggestion
      You can view all servers available in your preferred data center by clicking the **Datacenter selection** drop-down menu in the top right corner.
```

File: environmental-footprint/environmental-footprint/additional-content/environmental-footprint-calculator.mdx (hunk 6 above)
Comment (jcirinosclwy) [inline environmental-footprint/environmental-footprint/additional-content/environmental-footprint-calculator.mdx:26]:
```suggestion
Our Methodology is based on the [ADEME (French Government Agency for Ecological Transition) Product Category Rules (PCR) for Datacenter and Cloud services](https://librairie.ademe.fr/produire-autrement/6105-methodological-standard-for-the-environmental-assessment-of-datacenter-it-hosting-services-and-cloud-services.html), which provide a standardized framework for calculating and reporting the environmental impact of data centers and cloud services. These rules are designed to ensure consistency, transparency, and comparability in the assessment of environmental footprints across the industry.
```

File: tutorials/load-overflow-proxmox-and-elastic-metal/index.mdx (hunk 9 above)
Comment (jcirinosclwy) [inline tutorials/load-overflow-proxmox-and-elastic-metal/index.mdx:48]:
```suggestion
1. Log into the Proxmox web interface and select your server in the datacenter view. Then click **Network** in the menu.
```

File: tutorials/load-overflow-proxmox-and-elastic-metal/index.mdx (hunk 10 above)
Comment (jcirinosclwy) [inline tutorials/load-overflow-proxmox-and-elastic-metal/index.mdx:52]:
```suggestion
4. Go to the **Datacenter** view and click **Create Cluster**.
```

File: tutorials/load-overflow-proxmox-and-elastic-metal/index.mdx (hunk 11 above)
Comment (jcirinosclwy) [inline tutorials/load-overflow-proxmox-and-elastic-metal/index.mdx:56]:
```suggestion
8. Go to the Proxmox interface of your second Elastic Metal server and click **Cluster** to enter the cluster configuration from the Datacenter view of the machine.
```

File: tutorials/create-esxi-cluster-dedibox/index.mdx (hunk 7 above)
Comment (jcirinosclwy) [inline tutorials/create-esxi-cluster-dedibox/index.mdx:250]:
```suggestion
2. Click **Actions** > **New datacenter** to create a new data center. A data center is a group of machines.
```

File: tutorials/create-esxi-cluster-dedibox/index.mdx (hunk 8 above)
Comment (jcirinosclwy) [inline tutorials/create-esxi-cluster-dedibox/index.mdx:273]:
```suggestion
1. From the **Datacenter** dashboard in your vSphere web interface click **Actions** > **New Cluster**:
```

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**